packages/core/lib/combo/patch-build/randomizer.ts

```diff
@@ -720,7 +720,7 @@ const gameEntrances = (worldId: number, game: Game, logic: LogicResult) => {
   return padBuffer16(toU32Buffer(data));
 };
 
-const randomizerDungeonsBits = (worldId: number, logic: LogicResult): Buffer => {
+const randomizerDungeonsBits = (worldId: number, logic: LogicResult, settings: Settings): Buffer => {
```

Comment:
No need to pass the `settings`, they're accessible through the `logic` object, please use `logic.settings` instead within the function